llvm/lib/Passes/PassRegistry.def

```diff
@@ -136,6 +136,7 @@ MODULE_PASS("SPIRITTAnnotations", SPIRITTAnnotationsPass())
 MODULE_PASS("deadargelim-sycl", DeadArgumentEliminationSYCLPass())
 MODULE_PASS("sycllowerwglocalmemory", SYCLLowerWGLocalMemoryPass())
 MODULE_PASS("lower-esimd-kernel-attrs", SYCLFixupESIMDKernelWrapperMDPass())
+MODULE_PASS("sycl-propagate-aspects-usage", PropagateAspectsUsagePass())
```

Comment:
May be rename Pass with SYCL prefix for uniformity?

Reply:
Done in 96139d671baf3ea32672932b73d2dc5c0a56663b